content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,401 @@
+
+</WarningBox>
+
+<br />
+
+4. To redeploy the gateway, copy the following code snippet to your terminal and provide the gateway configuration file as input.
+
+
+```bash
+docker run -it –rm \
+–net-host \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /tmp:/opt/spectrocloud \
+gcr.io/spectro-images-public/release/spectro-installer:1.0.12 \
+-s true \
+-c /opt/spectrocloud/pcg.yaml
+```
+
+The installer requests available bare metal machines in your MAAS environment on which to install the gateway.
+
+If you need assistance, please visit our [Customer Support](https://spectrocloud.atlassian.net/servicedesk/customer/portals) portal.
+
+</Tabs.TabPane>
+
+</Tabs>
+
+<br />
+
+# Update and Manage the MAAS Gateway
+
+Palette maintains the Operating System (OS) image and all configurations for the PCG. Periodically, the OS images, configurations, and other components need to be updated to resolve security or functionality issues. Palette releases updates when required, and informs you with an update notification when you click on the gateway in the **Manage Cloud Gateways** page.
+
+Review the changes in the update notification, and apply the update when you are ready. 
+
+Updating the cloud gateway does not result in any downtime for the tenant clusters. During the update process, new cluster provisioning is unavailable. New cluster requests are queued and processed when the gateway update is complete.
+
+<br />
+
+# Delete the MAAS Gateway
+
+Follow these steps to delete a MAAS gateway.
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main menu** and select **Tenant Settings > Private Cloud Gateways**.
+
+
+3. Click the **three-dot Menu** for the gateway instance you want to delete and choose **Delete**.
+
+    Palette checks for running tenant clusters associated with the gateway instance and displays an error message if it detects any. 
+    <br />
+
+4. If there are running clusters, delete them and retry deleting the gateway instance.
+
+<br />
+
+# Resize the MAAS Gateway
+
+You can set up a PCG as a single-node (no HA) or three-node (HA) cluster. You can set up a PCG initially with one node and resize it to three nodes at a later time.
+
+<br />
+
+<InfoBox>
+
+For production environments, we recommend setting up three nodes.
+
+</InfoBox>
+
+
+
+Follow these steps to resize a single-node gateway to three nodes.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and select **Tenant Settings > Private Cloud Gateways**.
+
+
+3. Click the **three-dot Menu** for the gateway instance you want to resize and choose **Set number of nodes**.
+
+
+4. Change the number of nodes to 3.
+
+Two new nodes will be created in the cluster.
```

Comment:
How do we validate this? Hint validation section

Reply:
Added validation